### DISCUSSION TITLE
关于搜索框移动问题

### DISCUSSION THREAD

**Discussion by I-lemon521** [YunYouJun/hexo-theme-starry#119] [Q&A 疑难杂症] [answered]
搜索框功能打开后一直显示在左下角，怎么调整位置呀

**Comment by YunYouJun** [comment, marked as answer]
见文档此处。<https://yun.yunyoujun.cn/guide/third-party-support.html#%E6%9C%AC%E5%9C%B0%E6%90%9C%E7%B4%A2>

> 如果你发现本地正常，部署后点击搜索按钮却会跳转至页面底部，可能单纯只是缓存问题。
